### PR TITLE
avrogen instructions -> readme

### PR DESCRIPTION
**Pull request by mhowlett**
@edenhill

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,16 @@ public class Program
 }
 ```
 
+### AvroGen tool
+
+The Avro serializer and deserializer provided by `Confluent.Kafka.Avro` can be used with the `GenericRecord` class
```
Comment (edenhill) [inline README.md]: Maybe make Confluent.Kafka.Avro a link to the NuGet package?

File: README.md [hunk 2]
```diff
@@ -141,6 +141,16 @@ public class Program
...
+or with specific classes generated using the `avrogen` tool 
+(available [here](https://github.com/confluentinc/avro/releases/download/v1.7.7.4/avrogen.zip)). Usage:
+
+```
+dotnet /path/to/avrogen.dll -s your_schema.asvc .
```
Comment (edenhill) [inline README.md]: Do we need to explain what is being generated and how to include them in the project (if that's needed?)?

File: README.md (hunk 1 above)
Comment (mhowlett) [reply README.md]: people find this via their ide, and it's linked above, so i think no need.

File: README.md (hunk 2 above)
Comment (mhowlett) [reply README.md]: yep, this would be better, but this was just a quick pr to get a link to the tool up here since it's not easy to find otherwise.

**Pull request merged**